app/views/admin/notifications/_notification.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if check_read notification %>
+  <%= link_to notification_path(notification) do %>
+    <li class="noti-li not-read">
+      <%= notification.title %>
+      <%= notification.created_at.strftime(t"date_time_format") %>
+    </li>
+  <% end %>
+<% else %>
+  <%= link_to notification_path(notification) do %>
+    <li class="noti-li seen">
```

Comment:
code này chỉ khác về class của li thôi nhỉ 
rứa sao ko làm dạng ri
`<li class="noti-li <%=  check_read notification ? "not-read" : "seen" %>">`

Reply:
dạ vâng ạ